_posts/2019-02-14-customize.md

```diff
@@ -907,125 +909,112 @@ $ yq w -i $TARGET public.app.clientTitle "New Title"
+| `userdata-bbb_multi_user_pen_only=` | If set to `true`, the whiteboard will be multi-user only, meaning that all participants can annotate it freely | `false`       |
+| `userdata-bbb_presenter_tools=`     | If set to `false`, the client will display the presenter tools                                                 | `true`        |
+| `userdata-bbb_multi_user_tools=`    | If set to `true`, the client will display the multi-user tools                                                 | `false`       |
 
-If set to `true`, the user will be able to join the audio part of the meeting without needing to share a microphone (listening only, without speaking)
 
-* `userdata-bbb_outside_toggle_recording=true`
-(default value is `false`)
-(only applicable when the client is embedded in an iframe)
-If set to `true`, the recording functionality in the meeting will be able to be started/stopped from the "parent" web page -- the same page where the BigBlueButton client is embedded
+### Themeing & styling parameters
 
-* `userdata-bbb_outside_toggle_self_voice=true`
-(default value is `false`)
-(only applicable when the client is embedded in an iframe)
+| Parameter                        | Description                                                                                                          | Default value |
+| -------------------------------- | -------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_custom_style=`     | URL encoded string with custom CSS                                                                                   | none          |
+| `userdata-bbb_custom_style_url=` | This parameter acts the same way as `userdata-bbb_custom_style` except that the CSS content comes from a hosted file | none          |
 
-If set to `true`, the current user's audio will be able to be muted/unmuted from the "parent" web page -- the same page where the BigBlueButton client is embedded
+### Layout parameters
 
+| Parameter                                  | Description                                                                                                           | Default value |
+| ------------------------------------------ | --------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `userdata-bbb_auto_swap_layout=`           | If set to `true`, the client will display the ask for feedback screen on logout                                       | `false`       |
+| `userdata-bbb_hide_presentation=`          | If set to `true`, the presentation area will be minimized automatically                                               | `false`       |
+| `userdata-bbb_show_participants_on_login=` | If set to `true`, the client will start the process of joining the audio bridge automatically upon loading the client | `false`       |
```

Comment:
the description does not fit here

Reply:
Well spotted!

Reply:
I've added a fix for this. Thanks for pointing this out.